ADOPTERS.md

```diff
@@ -5,10 +5,13 @@ These adopters use the project either directly or as a reference for their own i
 
 | Organization | Details                                                                   | References                                                |
 | ------------ | ------------------------------------------------------------------------- | --------------------------------------------------------- |
+| [Mercado Libre][Meli]   | Mercado Libre is using this portal internally for discovery of InnerSource Projects | See the [known instances][known_instances] section of the Portal Pattern |
```

Comment:
```suggestion
| [Mercado Libre][Meli]   | Mercado Libre is using this portal internally for discovery of InnerSource Projects | See the [known instances][known_instances] section of the InnerSource Portal pattern. |
```

Reply:
I would spell out the full name of the pattern.

Even thinking if maybe the link would be better to place on the pattern name, than on the "known instances".
While the terminology "known instances" has a specific mean for the InnerSource Commons patterns, it is probably not that easy to understand for the users of the SAP portal. Even leaving that phrase away entirely could be an option?

Reply:
Naming the section helps to find the mention on the pattern page.

How about something like this?

`See the "known instances" section of the [InnerSource portal pattern](https://patterns.innersourcecommons.org/p/innersource-portal)